include/crm/crm.h

```diff
@@ -213,17 +216,6 @@ extern char *crm_system_name;
 
 typedef GList *GListPtr;
 
-#  include <crm/common/logging.h>
-#  include <crm/common/util.h>
-
-static inline const char *
-crm_action_str(const char *task, guint interval_ms) {
-    if(safe_str_eq(task, RSC_STATUS) && !interval_ms) {
-        return "probe";
-    }
-    return task;
-}
-
```

Comment:
Can't do ... this is public API (and I could see it being useful as such). Making it non-inline would still break compatibility. Probably the best alternative would be to use strcasecmp directly here (btw this is one of the cases where it should really be case-sensitive, but that needs to wait for a compatibility break).

Reply:
I thought that might be the case.  Well, switching to strcasecmp here will accomplish the same thing (eliminating a use of safe_str_eq in crm.h that makes the includes difficult) so that's fine.